src/content/1.7/development/webservice/tutorials/2-testing-access.md

```diff
@@ -0,0 +1,164 @@
+
+The shop should prompt you for a username and a password to enter. The username is the authentication key you created and **there is no password to enter**. If no permission has been set for the key, than the browser will keep asking you to enter the key indefinitely.
+
+The second and more appropriate way to access the API is to include your access key in the url, this will prevent you from entering any user name.
+This is also the recommended way to call the API from a javascript client, or any application. Here is an example, assuming your access API key is `UCCLLQ9N2ARSHWCXLT74KUKSSK34BFKX`:
+
+* At the root of the server: https://UCCLLQ9N2ARSHWCXLT74KUKSSK34BFKX@example.com/api/
+* In a subfolder of the server: https://UCCLLQ9N2ARSHWCXLT74KUKSSK34BFKX@example.com/prestashop/api/
+
+{{% notice note %}}
+To test/call your APIs we recommend you use an API client such as [Insomnia](https://insomnia.rest/) or [Postman](https://www.getpostman.com/), it is easier to call the APIs than with a browser, especially for write actions.
+{{% /notice %}}
+
+{{% notice warning %}}
+As you noticed no password nor authentication process is required to access the APIs which is why you need to be **extra careful** with your access key rights and how (and whose) you distribute them.
+{{% /notice %}}
+
+## Available resources
+
+The `/api/` URL gives you the root of all the resources, in the form of an XML file. Here is an example of what you should see if you authorized full access to `addresses`, `images` and `products` resources:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
+<api shopName="Prestashop">
+<addresses xlink:href="http://example.com/api/addresses" get="true" put="true" post="true" delete="true" head="true">
+<description xlink:href="http://example.com/api/addresses" get="true" put="true" post="true" delete="true" head="true">
+The Customer, Brand and Customer addresses</description>
+<schema xlink:href="http://example.com/api/addresses?schema=blank" type="blank"/>
+<schema xlink:href="http://example.com/api/addresses?schema=synopsis" type="synopsis"/>
+</addresses>
+<images xlink:href="http://example.com/api/images" get="true" put="true" post="true" delete="true" head="true">
+<description xlink:href="http://example.com/api/images" get="true" put="true" post="true" delete="true" head="true">
+The images</description>
+</images>
+<products xlink:href="http://example.com/api/products" get="true" put="true" post="true" delete="true" head="true">
+<description xlink:href="http://example.com/api/products" get="true" put="true" post="true" delete="true" head="true">
+The products</description>
+<schema xlink:href="http://example.com/api/products?schema=blank" type="blank"/>
+<schema xlink:href="http://example.com/api/products?schema=synopsis" type="synopsis"/>
+</products>
+</api>
+</prestashop>
```

Comment:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<prestashop xmlns:xlink="http://www.w3.org/1999/xlink">
  <api shopName="Prestashop">
    <addresses xlink:href="http://example.com/api/addresses" get="true" put="true" post="true" delete="true" head="true">
      <description xlink:href="http://example.com/api/addresses" get="true" put="true" post="true" delete="true" head="true">
      The Customer, Brand and Customer addresses</description>
      <schema xlink:href="http://example.com/api/addresses?schema=blank" type="blank"/>
      <schema xlink:href="http://example.com/api/addresses?schema=synopsis" type="synopsis"/>
    </addresses>
    <images xlink:href="http://example.com/api/images" get="true" put="true" post="true" delete="true" head="true">
      <description xlink:href="http://example.com/api/images" get="true" put="true" post="true" delete="true" head="true">
      The images</description>
    </images>
    <products xlink:href="http://example.com/api/products" get="true" put="true" post="true" delete="true" head="true">
      <description xlink:href="http://example.com/api/products" get="true" put="true" post="true" delete="true" head="true">
      The products</description>
      <schema xlink:href="http://example.com/api/products?schema=blank" type="blank"/>
      <schema xlink:href="http://example.com/api/products?schema=synopsis" type="synopsis"/>
    </products>
  </api>
</prestashop>
```

Reply:
I didn't indent the XML because the API returns it this way so I preferred to stick with the real way rather than indent it You think it's better to pretty display it?

Reply:
Yeah, it's more readable. Clients like postman pretty print xml. Chrome does it as well when it's sent with the appropriate headers.